### DISCUSSION TITLE
Primeng tabview tabs selected

### DISCUSSION THREAD

**Discussion by mervetagci** [primefaces/community#881] [PrimeNG] [answered]
Hello. 
In the closable section of [prime ng's tabview](https://primeng.org/tabview) documentation, I see that when I remove the last tab, it goes to the first tab. My question is: How can I make it go to the tab on the left instead of the first tab?
(I guess it's a situation where tabIndex takes values ​​of 0 and -1). When changing the TabIndex value, is there a way to give the tab to the left of the closed tab instead of the first tab? (
Sample behavior should be as follows: Let there be tabs 1, 2 and 3. When I close number 3, I want tab number 2 to be shown). Thanks in advance.

![Recording20220807at210932](https://github.com/primefaces/community/assets/66209345/20b2a35e-686b-4e21-8c07-276b70b07525)
I want to do same like this but i dont know actually how its working?

**Comment by emiCacc** [comment]
Hello! 
Have you tried to save the value of the last element? For example you can create a variable 'lastElement' and save that value there, that way you will have as a reference your last value and you can use it whenever you need it (even in future elements of your project).

For your example you should only use that variable in the current card and when you close it (that is, when you call your delete method) you can do a custom focus event for example returnFocus() to your lastElement.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Please see the simple example, further logic is up to you I've just prepared it to demonstrate that it's possible but I'm not sure if it needs additional tweaks in the logic. So please customize it to fit your needs.

https://stackblitz.com/edit/ksyyv9?file=src%2Fapp%2Fdemo%2Ftab-view-basic-demo.html,src%2Fapp%2Fdemo%2Ftab-view-basic-demo.ts
